Build fail when trying to launch extension from debugger

Please *thumbs-up* 👍 this issue if it personally affects you! You can do this by clicking on the emoji-face on the top right of this post. Issues with more thumbs-up will be prioritized.

-----

### What did you do?
Forked vim/master, cloned it and tried to build.
1. ```npm install```
2. ```gulp```
3. run "Launch Extension" from vscode debugger

### What did you expect to happen?
build to be successful and launch extension

### What happened instead?
build failed

### Technical details:
```
[23:56:06] Using gulpfile ~/Documents/Vim/gulpfile.js
[23:56:06] Starting 'tslint'...
[23:56:06] Starting 'compile'...
> vim@0.5.3 vscode:prepublish /Users/soojae/Documents/Vim
> node ./node_modules/vscode/bin/compile -p ./
src/actions/actions.ts(1141,53): error TS2532: Object is possibly 'undefined'.
src/actions/actions.ts(1141,68): error TS2532: Object is possibly 'undefined'.
src/actions/actions.ts(1147,71): error TS2532: Object is possibly 'undefined'.
src/textEditor.ts(205,30): error TS2532: Object is possibly 'undefined'.
src/textEditor.ts(234,30): error TS2532: Object is possibly 'undefined'.
src/textEditor.ts(235,65): error TS2532: Object is possibly 'undefined'.
src/textEditor.ts(238,52): error TS2532: Object is possibly 'undefined'.
npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "vscode:prepublish"
npm ERR! node v7.6.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! vim@0.5.3 vscode:prepublish: `node ./node_modules/vscode/bin/compile -p ./`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vim@0.5.3 vscode:prepublish script 'node ./node_modules/vscode/bin/compile -p ./'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the vim package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./node_modules/vscode/bin/compile -p ./
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs vim
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls vim
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /Users/soojae/Documents/Vim/npm-debug.log
[23:56:18] 'compile' errored after 12 s
[23:56:18] Error: Command `npm run vscode:prepublish` failed with exit code 1
    at ChildProcess.exithandler (child_process.js:211:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:192:7)
    at maybeClose (internal/child_process.js:890:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:189:7)
    at Pipe._handle.close [as _onclose] (net.js:501:12)
[23:56:25] Finished 'tslint' after 19 s
```
* VSCode Version: 1.9.1
* VsCodeVim Version: 0.5.3
* OS: macOS Sierra 10.12.3

--- a/src/configuration/configuration.ts
+++ b/src/configuration/configuration.ts
@@ -145,22 +145,22 @@ class ConfigurationClass {
    * Size of a tab character.
    */
   @overlapSetting({ codeName: "tabSize", default: 8})
-  tabstop: number | undefined = undefined;
+  tabstop: number;
 
   /**
    * Use spaces when the user presses tab?
    */
   @overlapSetting({ codeName: "insertSpaces", default: false})
-  expandtab: boolean | undefined = undefined;
+  expandtab: boolean;
 
   @overlapSetting({ codeName: "lineNumbers", default: true, codeValueMapping: {true: "on", false: "off"}})
-  number: boolean | undefined = undefined;
+  number: boolean;
 
   /**
    * Show relative line numbers?
    */
   @overlapSetting({ codeName: "lineNumbers", default: false, codeValueMapping: {true: "relative", false: "off"}})
-  relativenumber: boolean | undefined = undefined;
+  relativenumber: boolean;
 
   iskeyword: string = "/\\()\"':,.;<>~!@#$%^&*|+=[]{}`?-";
 }

